Expose individual host's install disk variable in inventory

akrzos opened the pull request.

akrzos commented:
@Noreen21 Could you try this with SNO and see if I broke anything with that install path?  @sferlin could you give this a review and maybe a test run too? Thanks!

akrzos commented:
@ebattat Added you to check out if ibmcloud still works here as well.

Noreen21 commented:
Will give it a shot after I am done with my current testing which should be completed by this week. :)

ebattat commented on ansible/roles/create-inventory/defaults/main/storage.yml:
```diff
@@ -0,0 +1,27 @@
+# Jetlag storage related vars
+
+# Default install disks for bm/rwn
+control_plane_install_disk: /dev/sda
```
Two generic comments:
1. OCP 4.13.rc(RHEL9) and on required disk by id and not support symbolic link anymore, so I think we should recommend to use it.
Do we want to mention in the comment the template for disk by wwn id:  /dev/disk/by-id/wwn-xxx

2. I recommend that the parameter should contains hint for bootable disk because the installation failed when it was a regular disk

install_disk => install_bootable_disk or bootable_disk

akrzos replied:
Good suggestions, I think we should put a reference in the sample inventory file since these vars actually affect more than one machine so you still have to go back and manually edit your inventory file to add the correct by-id address per machine anyhow

sferlin reviewed (commented):
Tested on bm OCP 4.13 and sno OCP 4.12.

Need to add a small fix to ansible/roles/wait-hosts-discovered/tasks/set_node_install_disk.yml

From hostvars[bm_node.requested_hostname].install_disk to hostvars[sno_cluster.json.hosts[0].requested_hostname | split('.') | first].install_disk

akrzos commented:
@sferlin I adjusted the sno-wait-hosts-discovered tasks to what you identified.

@ebattat I added a comment to address the fact for ocp 4.13 to use by-id reference instead of symbolic reference.

Lets get this merged so we can start working on part 2 to solve ocp 4.13 installs

ebattat commented:
@akrzos, Updating that OCP 4.13.0-rc4 installed successfully using Jetlag by [ Benchmark-Runner CI  ](https://github.com/redhat-performance/benchmark-runner/actions/runs/4766274965)

ebattat reviewed (approved):
/LGTM

Noreen21 reviewed (approved):
LGTM! Thanks Alex!
